### PR TITLE
CameraRoll Support (Bug Fix #123)

### LINKED ISSUE #123: CameraRoll Support (Bug)
I'm used version
pod 'WeScan', :git => 'https://github.com/WeTransfer/WeScan.git'

then when I start scan image which picked from camera roll, I got wrong detected rectangle.
How can I fix that?

photo_d10405_0128072031788

### PR DESCRIPTION
**Pull request by xiaotaner**

### CODE REVIEW

**Comment by xiaotaner** [comment]
@justJS of course，you can add it back

**Comment by xiaotaner** [comment]
@justJS , I think line 90 and 91 is need to be removed, because scanning rectangle from camera roll and camera is synchronous after the bellow change:
image

**Comment by justJS** [comment]
It may be but the request handler is still a completion block. If you try to return from within it you’ll see an error as you’re not returning from the enclosing function.

**Comment by xiaotaner** [comment]
I think you are right , you can add it back

**Comment by xiaotaner** [comment]
I am so sorry, because i do not know show to write test --!
You can edit my code or you can commit a new PR

File: WeScan/ImageScannerController.swift [hunk 1]
```diff
@@ -84,15 +84,12 @@ public final class ImageScannerController: UINavigationController {
             if #available(iOS 11.0, *) {
                 // Use the VisionRectangleDetector on iOS 11 to attempt to find a rectangle from the initial image.
                 VisionRectangleDetector.rectangle(forImage: ciImage) { (quad) in
-                    detectedQuad = quad
+                    detectedQuad = quad?.toCartesian(withHeight: image.size.height)
                     detectedQuad?.reorganize()
-
-                    let editViewController = EditScanViewController(image: image, quad: detectedQuad, rotateImage: false)
-                    self.setViewControllers([editViewController], animated: true)
```
Comment (AvdLee) [inline WeScan/ImageScannerController.swift]: @Boris-Em do you know whether this is OK? it seems like a breaking change to me

Comment (justJS) [reply WeScan/ImageScannerController.swift]: This code should not be removed as the `defer` block is *not* called - Vision uses escaping closures.

**Comment by AvdLee** [comment]
Closing this one for inactivity. Also, will be fixed with https://github.com/WeTransfer/WeScan/pull/165

**Pull request closed without merging**